server/schema/dashboard.go

```diff
@@ -54,6 +55,7 @@ type DashboardElementView struct {
 
 	Realtime types.NullInt64   `json:"realtime"`
 	Servers  []newrelic.Server `json:"servers,omitempty"`
+	Tasks    []*teamcity.Build `json:"tasks,omitempty"`
```

Comment:
why pointer? and if you use pointer here, why you didn't change `newrelic.Server` to `*newrelic.Server`?  
it seems to be same things, so it whould be nice if they will be implemented in the same way